Server/Admin/routers/admin_chatbot.py

```
@@ -852,4 +1033,57 @@ def filter_query(query, model):
         # TODO: Implement Excel report generation
         raise HTTPException(status_code=501, detail="Excel report generation not implemented yet.")
     else:
-        raise HTTPException(status_code=400, detail="Invalid report format.") 
\ No newline at end of file
+        raise HTTPException(status_code=400, detail="Invalid report format.")
+
+
+@admin_chatbot_router.get("/export-users-excel")
+async def export_users_excel(
+    authorization: str = Header(None),
+    db: Session = Depends(get_db)
```

Ruff (0.14.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)